wiki/Brand_identity_guidelines/en.md

```diff
@@ -0,0 +1,69 @@
+# Brand identity guidelines
+
```

Comment:
I suggest adding a opening statement here so people can understand what's the purpose of this Press Kit.